src/main/resources/application.yaml

```diff
@@ -20,6 +20,33 @@ spring:
       STRICT_DUPLICATE_DETECTION: true
   application:
     name: Account Management
+  datasource:
+    driver-class-name: org.postgresql.Driver
+
+    url: jdbc:postgresql://${DB_NAME}.postgres.database.azure.com:${DB_PORT}/postgres
+    username: ${DB_USER}
+    password: ${DB_PASS}
+    properties:
+      charSet: UTF-8
+    #    hikari:
```

Comment:
Should this be here / commented out?

Reply:
nah not needed atm, left it in to match the other services, but ill chuck it out